library.c

```diff
@@ -2968,7 +2968,7 @@ redis_unpack(RedisSock *redis_sock, const char *val, int val_len, zval *z_ret)
 #ifdef HAVE_REDIS_ZSTD
             {
                 char *data;
-                size_t len;
+                unsigned long long len;
```

Comment:
From zstd.h

```
#define ZSTD_CONTENTSIZE_UNKNOWN (0ULL - 1)
#define ZSTD_CONTENTSIZE_ERROR   (0ULL - 2)
ZSTDLIB_API unsigned long long ZSTD_getFrameContentSize(const void *src, size_t srcSize);

```

So we have to use an `unsigned long long` (perhaps a cast `(size_t)ZSTD_CONTENTSIZE_UNKNOWN` may also work)